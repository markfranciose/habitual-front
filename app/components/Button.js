import React, { Component } from 'react';
import { Text, StyleSheet } from 'react-native';

export default class HabitButton extends Component {
  render() {
    return(
        <Text>There should be text in here!!!</Text>
      )
  }
}

const styles = StyleSheet.create({
  button: {
    borderRadius: 10,
    borderColor: 'white',
  },
});
