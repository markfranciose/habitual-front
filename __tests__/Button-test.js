import 'react-native';
import React from 'react';
import Button from '../app/components/Button'

import renderer from 'react-test-renderer';

test ('renders correctl', () => {
  const tree = renderer.create(
    <Button />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
