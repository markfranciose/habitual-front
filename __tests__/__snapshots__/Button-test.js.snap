// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctl 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
>
  There should be text in here!!!
</Text>
`;
